src/components/Reports/SalaryReport/SalaryReportTable.tsx

```diff
@@ -1,10 +1,238 @@
-import React from 'react';
-import { Box } from '@material-ui/core';
+/* eslint-disable import/no-unresolved */
+import React, { useEffect, useState } from 'react';
+import _ from 'lodash';
+import clsx from 'clsx';
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  colors,
+  Grid,
+  SvgIcon,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import CodeIcon from '@material-ui/icons/Code';
+import PrintIcon from '@material-ui/icons/Print';
+import { CSVLink } from 'react-csv';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import {
+  FourteenMonthReportContact,
+  FourteenMonthReportCurrencyType,
+  // eslint-disable-next-line import/extensions
+} from 'graphql/types.generated';
+// eslint-disable-next-line import/extensions
+import { useFourteenMonthReportQuery } from 'pages/accountLists/[accountListId]/reports/graphql/GetReportFourteenMonth.generated';
 
 interface Props {
+  className?: string;
   accountListId: string;
+  title: string;
 }
 
-export const SalaryReportTable: React.FC<Props> = () => {
-  return <Box />;
+const useStyles = makeStyles(() => ({
+  root: {},
+  downloadCsv: {
+    color: 'inherit',
+    textDecoration: 'none',
+  },
+}));
+
+export const SalaryReportTable: React.FC<Props> = ({
+  className,
+  accountListId,
+  title,
+  ...rest
+}) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const [contacts, setContacts] = useState<FourteenMonthReportContact[]>([]);
+
+  const { data, loading, error } = useFourteenMonthReportQuery({
+    variables: {
+      accountListId,
+      currencyType: FourteenMonthReportCurrencyType.Salary,
+    },
+  });
+
+  const salaryCurrency = data?.fourteenMonthReport.salaryCurrency;
+  const currencyGroups = data?.fourteenMonthReport.currencyGroups;
+
+  useEffect(() => {
+    if (currencyGroups) {
+      setContacts(
+        currencyGroups.flatMap((currencyGroup) => [...currencyGroup.contacts]),
+      );
+    }
+  }, [currencyGroups]);
+
+  const renderLoading = () => (
+    <TableRow>
+      <TableCell colSpan={16}>
+        <Box
+          height="100%"
+          alignItems="center"
+          justifyContent="center"
+          bgcolor={colors.green[600]}
+        >
+          Loading
+        </Box>
+      </TableCell>
+    </TableRow>
+  );
+
+  const renderEmpty = () => (
+    <TableRow>
+      <TableCell colSpan={16}>No Data</TableCell>
```

Comment:
Can you make this somewhat match the existing codebase?
<img width="1439" alt="Screen Shot 2021-06-18 at 4 30 14 PM" src="https://user-images.githubusercontent.com/756501/122624210-fc60d700-d053-11eb-9db2-3f6e50ec627f.png">
Some sort of icon (doesn't have to be that exact one), the message text, and the two action buttons. For "Connect Services" you can use the handoff link component since I don't think we're going to get to rewriting the settings section for a while. I created https://jira.cru.org/browse/MPDX-7053 for the add donations modal so you can just add a placeholder button for now.

Reply:
@OzzieOrca There was already the component so I just updated a little bit.